main.js

```diff
@@ -1,6 +1,25 @@
 
 var buttonRow = document.getElementById("buttonRow")
+var category = '';
 
 buttonRow.addEventListener("click", function(event){
   event.preventDefault();
+  category = event.target.id
 })
+
+
+
+var minutes = document.getElementById("minutes");
+var seconds = document.getElementById("seconds");
+var startActivityButton = document.getElementById("startActivityButton");
+var description = document.getElementById("descriptionInput")
+
+startActivityButton.addEventListener("click", handleTime);
+
+function handleTime(event) {
+  event.preventDefault(event);
+  if(minutes.value.includes("e")){
```

Comment:
Might want to revisit the 'e' piece to ensure it's not pushing to the new object